mlir/test/Dialect/XeGPU/xegpu-wg-to-sg-rr.mlir

```diff
@@ -0,0 +1,74 @@
+// RUN: mlir-opt --xegpu-wg-to-sg-distribute -split-input-file %s | FileCheck %s
+
+gpu.module @test_round_robin_assignment {
+  // CHECK-LABEL: test_create_nd_tdesc
+  // CHECK-SAME: %[[ARG_0:.*]]: memref<24x32xf32>
+  gpu.func @test_create_nd_tdesc(%src: memref<24x32xf32>) {
+      // CHECK-COUNT-12: xegpu.create_nd_tdesc %[[ARG_0]][%{{.*}}, %{{.*}}] : memref<24x32xf32>
+      // CHECK-SAME: -> !xegpu.tensor_desc<2x2xf32, #xegpu.layout<lane_layout = [2, 2], lane_data = [1, 1]>>
```

Comment:
One issue with count check is that this test will also pass fine for, for example, `COUNT-10`
The guard against this, it could be good to add `CHECK-NOT: OP` e.g., `// CHECK-NOT: xegpu.create_nd_tdesc`, as the last line to ensure there aren't more ops than expected

Reply:
added